src/main/resources/application-local.yaml

```diff
@@ -25,4 +25,8 @@ eidservice:
     keystore: classpath:testKeys/sign.localhost_2382.p12
     type: PKCS12
     alias: sign.localhost
-    password: 2382
\ No newline at end of file
+    password: 2382
+
+csp:
+  defaultConfig: "default-src 'self';script-src 'self';style-src 'self';font-src 'self';img-src 'self';connect-src 'self';"
+  frameAncestors: frame-ancestors 'self' localhost;
```

Comment:
The reason of the separation is to avoid mistake when editing a long string character